source/ddbus/router.d

```diff
@@ -66,95 +67,120 @@ class MessageRouter {
     MessageType type = msg.type();
-    if(type != MessageType.Call && type != MessageType.Signal)
+    if (type != MessageType.Call && type != MessageType.Signal) {
       return false;
+    }
+
     auto pattern = MessagePattern(msg);
     // import std.stdio; debug writeln("Handling ", pattern);
 
-    if(pattern.iface == "org.freedesktop.DBus.Introspectable" &&
-      pattern.method == "Introspect" && !pattern.signal) {
+    if (pattern.iface == "org.freedesktop.DBus.Introspectable"
+        && pattern.method == "Introspect" && !pattern.signal) {
       handleIntrospect(pattern.path, msg, conn);
       return true;
     }
 
     MessageHandler* handler = (pattern in callTable);
-    if(handler is null) return false;
+    if (handler is null) {
+      return false;
+    }
 
     // Check for matching argument types
-    version(DDBusNoChecking) {
+    version (DDBusNoChecking) {
 
     } else {
-      if(!equal(join(handler.argSig), msg.signature())) {
+      if (!equal(join(handler.argSig), msg.signature())) {
         return false;
       }
     }
 
-    handler.func(msg,conn);
+    handler.func(msg, conn);
     return true;
   }
 
   void setHandler(Ret, Args...)(MessagePattern patt, Ret delegate(Args) handler) {
     void handlerWrapper(Message call, Connection conn) {
       Tuple!Args args = call.readTuple!(Tuple!Args)();
       auto retMsg = call.createReturn();
-      static if(!is(Ret == void)) {
+
+      static if (!is(Ret == void)) {
         Ret ret = handler(args.expand);
-        static if (is(Ret == Tuple!T, T...))
+        static if (is(Ret == Tuple!T, T...)) {
           retMsg.build!T(ret.expand);
-        else
+        } else {
           retMsg.build(ret);
+        }
       } else {
         handler(args.expand);
       }
-      if(!patt.signal)
+
+      if (!patt.signal) {
         conn.send(retMsg);
+      }
     }
+
     static string[] args = typeSigArr!Args;
-    static if(is(Ret==void)) {
+
+    static if (is(Ret == void)) {
       static string[] ret = [];
     } else {
       static string[] ret = typeSigReturn!Ret;
     }
-    MessageHandler handleStruct = {func: &handlerWrapper, argSig: args, retSig: ret};
+
+    // dfmt off
+    MessageHandler handleStruct = {
+      func: &handlerWrapper,
+      argSig: args,
+      retSig: ret
+    };
+    // dfmt on
+
     callTable[patt] = handleStruct;
   }
 
   static string introspectHeader = `<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
 <node name="%s">`;
 
   string introspectXML(string path) {
-    auto methods = callTable.byKey().filter!(a => (a.path == path) && !a.signal)().array()
-      // .schwartzSort!((a) => a.iface, "a<b")();
-      .sort!((a,b) => a.iface < b.iface)();
+    auto methods = callTable.byKey().filter!(a => (a.path == path) && !a.signal)().array() // .schwartzSort!((a) => a.iface, "a<b")();
+    .sort!((a,
+        b) => a.iface < b.iface)();
```

Comment:
the comment breaks this formatting. If anyone remembers what that schwartzSort did (@trishume) maybe explain why you commented it instead of removing it.

Maybe put this in a `//dfmt off` block and put each functional call in a separate line.